### DISCUSSION TITLE
[Question] Can't download, Crash in 0.70

### DISCUSSION THREAD

**Discussion by Mofu0406** [kubilus1/autoortho#434] [Q&A]
### Please check existing docs and issues first.

- [X] I have checked for existing answers
- [X] This is not a bug or other "Issue"
- [X] I have read the FAQ and documentation

### System Details

- OS: Windows 11
- XPlane Version: X-Plane 12
- AutoOrtho Version 0.70
- Filesystem type
Tried both Dokan and Winfsp but same issue

### question

**Describe your question**
The downloading speed is 0.2mbps to 1.5mbps and my WiFi usually downloads at 100mbps.
I waited until the download ends, however it stopped downloading and it eventually crashed.

If anybody know how to fix the problem, please tell me.
Thank you.

UPDATE: I've installed 0.6 instead, then it is downloading with no problems at 10mbps.
![image](https://github.com/kubilus1/autoortho/assets/130621536/0f975990-0176-4ad7-9c60-9960328628d5)

**Comment by Jeff2122** [comment]
There are many reporting that v0.7.0 has some problems with the installation process. Revert to v0.6.1 to do your install, then go back to v0.7.0 to run it.
